drools-wb-webapp/src/main/java/org/drools/workbench/client/DroolsWorkbenchEntryPoint.java

```diff
@@ -55,16 +54,18 @@
     protected DefaultAdminPageHelper adminPageHelper;
 
     @Inject
-    public DroolsWorkbenchEntryPoint( final Caller<AppConfigService> appConfigService,
-                                      final Caller<PlaceManagerActivityService> pmas,
-                                      final ActivityBeansCache activityBeansCache,
-                                      final PlaceManager placeManager,
-                                      final SyncBeanManager iocManager,
-                                      final DefaultWorkbenchFeaturesMenusHelper menusHelper,
-                                      final WorkbenchMenuBarPresenter menuBar,
-                                      AdminPage adminPage,
-                                      DefaultAdminPageHelper adminPageHelper) {
-        super( appConfigService, pmas, activityBeansCache );
+    public DroolsWorkbenchEntryPoint(final Caller<AppConfigService> appConfigService,
+                                     final Caller<PlaceManagerActivityService> pmas,
+                                     final ActivityBeansCache activityBeansCache,
+                                     final PlaceManager placeManager,
+                                     final SyncBeanManager iocManager,
```

Comment:
Hi @paulovmr, just question, is the `iocManager` still needed after your changes? Is `iocManager` used in some child class of `DroolsWorkbenchEntryPoint`?